src/pymmcore_widgets/useq_widgets/points_plans/_points_plan_widget.py

```diff
@@ -52,15 +76,49 @@ def __init__(
         self._well_view.maxPointsDetected.connect(self._on_view_max_points_detected)
         self._well_view.positionClicked.connect(self._on_view_position_clicked)
 
-        if plan is not None:
-            self.setValue(plan)
+        # if plan is not None:
+        self.setValue(plan or INIT_PLAN)
+
+        # init the view with the current well size
+        self.setWellSize(self._well_width_µm or 6, self._well_height_µm or 6)
```

Comment:
isn't `_well_width_µm` in microns and `6` in mm?

Reply:
yes 👍🏻, I changed that in [1c03208](https://github.com/pymmcore-plus/pymmcore-widgets/pull/319/commits/1c0320808f928c183296071f89d59f5898bd2aa3)